eng/pipelines/dotnet-sqlclient-ci-core.yml

```diff
@@ -41,6 +41,18 @@ parameters:
   type: object
   default: [false, true]
 
+- name: SNIVersion
+  displayName: |
+    Use a specific SNI version from the internal validation feed
+  type: string
+  default: ''
+
+- name: SNIValidationFeed
+  displayName: |
+    The specific SNI test version is pulled from this feed
```

Comment:
This isn't a great display name for the parameter... I'd go for something like "SNI NuGet Feed".